cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -33,21 +31,38 @@ DESCRIPTION
                Department department;
            |};
 
-       Then, generate the client API and SQL script based on the data model. Use the generated
-       API to query and manipulate the persistent data in the application and use the generated
-       SQL script to create the database tables.
+       Then, generate the client API based on the data model. You can use the `bal build` command to generate the client.
+       This process regenerates the client when modifying the data model during project builds.
+       Use the generated API to query and manipulate the persistent data in the application.
        For example:
 
-       $ bal persist generate
+       $ bal build
+
+       Apart from the above commands, the following commands are also available:
+
+       $ bal persist init
+
+       This command initializes the package for persistence.
+       This command will create a new directory named 'persist' in the project directory. The 'persist' directory will
+       contain empty model definition file which can be used to define the data model of the Ballerina project.
+
+       $ bal persist generate --module persist --datastore mysql
+
+       This command generates the client API based on the data model defined in the "persist" directory. Usually used
+       for one time generation of the client.
```

Comment:
```suggestion
       for one-time generation of the client API.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/commit/ef73dd6fdbd23ed185882b2a23767ffaa7c67725